rpm/xulrunner-qt5.spec

```
@@ -331,6 +338,12 @@ done
 
 %build
 
+# Move the .git directory out of the way as cargo gets confused and thinks it
+# needs to update our submodule.
+%if %{with git_workaround}
+%__mv %_builddir/.git %_builddir/.git-disabled ||:
```

If nothing else is really possible to prevent cargo doing that, then I'd suggest at least

```suggestion
trap '%__mv %_builddir/.git-disabled %_builddir/.git ||:' EXIT
%__mv %_builddir/.git %_builddir/.git-disabled ||:
```

:) and omit the restoration below.

---

Isn't it possible to disable that with `.cargo/config.toml`? https://doc.rust-lang.org/cargo/reference/config.html#netoffline

```
[net]
offline = true
```

---

We already have echo

> "export CARGO_NET_OFFLINE=1" >> "%BUILD_DIR"/rpm-shared.env

Which should be effectively the same.